src/ptf/testutils.py

```diff
@@ -3179,7 +3179,7 @@ def ptf_ports(num=None):
 
 
 def port_to_tuple(port):
-    if type(port) is int or (sys.version_info[0] == 2 and type(port) is int):
```

Comment:
this one must have been generated by some 2-to-3 tool...

Reply:
I looked at the git commit history on this one, since it looked so strange.  At one point it was less weird looking, I think something like `is long` in the last part instead of `is int`.